packages/web-app-importer/src/extensions.ts

```diff
@@ -51,6 +52,9 @@ export const extensions = ({ applicationConfig }: ApplicationSetupOptions) => {
   })
 
   const handler = async () => {
+    const currentThemeName = useLocalStorage('oc_currentThemeName', '')
+    const renderDarkTheme = unref(currentThemeName) === themeNameDark
```

Comment:
I feel like we could add a composable in `web-pkg` for the light/dark-mode related things? It could return `currentTheme` as a `computed` and also export the theme names. Then you don't need to import from runtime.

Reply:
not possible currently due to fuckups